### ISSUE TITLE
Allow adding alt text to images

### ISSUE THREAD

**Issue by mayhair** [db0/pythorhead#106]
Since [Lemmy v0.19.4](https://join-lemmy.org/news/2024-06-07_-_Lemmy_Release_v0.19.4_-_Image_Proxying_and_Federation_improvements), it is now possible to add alt text to image posts. This is very useful for providing descriptions of image posts that are accessible by blind or visually impaired people, while not intruding into the post body. However, it seems Pythorhead does not yet have a way to manipulate alt text. Could this be remedied in future versions? (Alt text should be usable not only for directly uploaded image posts, but also for those using external links.)

[Lemmy GitHub issue for alt text, links to corresponding commits](https://github.com/LemmyNet/lemmy/issues/1086)

**Comment by chris-y** [comment]
Just seen this - I added this in #114 so I think this can be closed?
